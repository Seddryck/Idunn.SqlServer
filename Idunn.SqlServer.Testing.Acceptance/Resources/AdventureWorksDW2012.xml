<?xml version="1.0" encoding="utf-8" ?>
<principal name="COLUMBIA\\cedri">
  <database server=".\sql2016" name="AdventureWorksDW2012">
    <permission name="CONNECT"/>
    <securable type="schema" name="dbo">
      <permission name="SELECT"/>
      <permission name="UPDATE"/>
    </securable>
  </database>
</principal>