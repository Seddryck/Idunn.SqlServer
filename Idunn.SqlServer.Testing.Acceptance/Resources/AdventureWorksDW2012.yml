principal:
  database:
    server: .\sql2016
    name: AdventureWorksDW2012
    permission: CONNECT
    securable: 
      type: schema 
      name: dbo
      permissions: 
        - SELECT
        - UPDATE