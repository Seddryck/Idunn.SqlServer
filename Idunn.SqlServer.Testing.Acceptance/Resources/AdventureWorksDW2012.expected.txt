Start execution ...
  Success: the permission CONNECT was granted on DATABASE::AdventureWorksDW2012 for principal dbo
  Success: the permission SELECT was granted on schema::dbo for principal dbo
  Success: the permission UPDATE was granted on schema::dbo for principal dbo
End of execution.
